### ISSUE TITLE
Galaxy S4 Mini - Android 4.4.2 - Images distorted

### ISSUE THREAD

**Issue by m2kk** [commonsguy/cwac-camera#271]
Hey,

I integrated cwac camera module into my app, and images on above device (Galaxy S4 Mini - Android 4.4.2) are distorted, and advice, or is it already an issue, or can it be easily fixed?
* Sample image attached.
![20150408024647](https://cloud.githubusercontent.com/assets/5787426/7051461/e4f5f84e-de28-11e4-9ae6-e6a7402f6f6d.jpg)

On any other device it worked out of the box as implemented.

Cheers,
Mike

**Comment by commonsguy** [comment]
I assume by "distorted", you are referring to the vertical stretch effect. Some questions:

- Is the photo coming out with the resolution that you are asking for?

- Is this photo being taken in portrait mode?

- Do you get the same results with the demo app? Or is this unique to your app?

- What is the `Build.PRODUCT` value for your device? Samsung has this habit of having several models with the same name like "Galaxy S4 Mini"...

**Comment by m2kk** [comment]
1) i cannot answer yet, but i'll ask
2) yes - but not sure, will ask too
3) not tested. should i?
4) i'll ask
thanks for the fast reply

**Comment by commonsguy** [comment]
> 3) not tested. should i?

Yes. If this problem is seen with the demo app, then it is clearly some incompatibility between my library and the device. If the problem cannot be seen with my app, then your code is somehow contributing to the problem. That may still mean the problem in the end is somewhere on my side, but I'd also then need a test app that can demonstrate the problem, as I won't be able to replicate it myself.

**Comment by m2kk** [comment]
ok, my app is there, if you have an s4 mini :) you're free to go:
https://play.google.com/store/apps/details?id=net.mikekober.myMory
i'll deploy the test app and ask the girl to do an image there too :) thx (and sure, stupid question)

**Comment by commonsguy** [comment]
> my app is there

That won't help me, as I won't have any idea what it is in your *code* that might be contributing to the problem. By "test app", I meant something with source code, which therefore probably isn't this app. My apologies for not being clearer on this in my earlier comment.

> if you have an s4 mini :) you're free to go:

I don't. If I have some confidence that I will be able to reproduce the problem, and once I know the `Build.PRODUCT` value so I know what the right hardware is, I can buy one. But if I'm not going to be able to reproduce the problem with the demo app, buying the hardware won't do me any good.

**Comment by m2kk** [comment]
thanks for replying, no need to buy one so far, hopefully tomorrow we get some more details.
